api/wikis/serializers.py

```diff
@@ -99,6 +108,40 @@ class NodeWikiSerializer(WikiSerializer):
         related_view_kwargs={'wiki_id': '<_id>'},
     )
 
+    def update(self, instance, validated_data):
+        auth = Auth(self.context['request'].user)
+        new_page_name = validated_data.get('page_name')
+        validated_data.pop('content', None)
+
+        try:
+            instance = instance.node.rename_node_wiki(instance.page_name, new_page_name, auth)
+        except (
+            NameInvalidError, NameMaximumLengthError,
+            PageCannotRenameError
+        ) as err:
+            raise ValidationError(err.message)
```

Comment:
Are you sure that `err.message` is an attribute? `Exception` objects don't have a `message` attribute unless it's explicitly set on them.

Reply:
Yep. `NameInvalidError`, `NameMaximumLengthError`, and `PageCannotRenameError` all set message.

https://github.com/CenterForOpenScience/osf.io/blob/develop/addons/wiki/models.py#L42
https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L2811

Although it doesn't hurt to add a check in case the messages are removed. Is there a cleaner way than:
```
...
except WikiError as err:
    if getattr(err, 'message'):
        raise ValidationError(err.message)
    raise ValidationError
```